config/ember-try.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-env node */
+'use strict';
+
+const getChannelURL = require('ember-source-channel-url');
+
+module.exports = function() {
+	return Promise.all([
+		getChannelURL('release'),
```

Comment:
Why did you remove the LTS ones?

Reply:
This is exactly the same file that we've in other addons like [ember-cli-workbox](https://github.com/BBVAEngineering/ember-cli-workbox/blob/79c634ed68197ef8adb76fa3fe45a8275ffc1c0a/config/ember-try.js), and comes from  ember-cli generator.